src/object.d

```diff
@@ -2878,12 +2878,12 @@ unittest
     assert(postblitRecurseOrder == order);
 }
 
-/++
-    Destroys the given object and puts it in an invalid state. It's used to
-    _destroy an object so that any cleanup which its destructor or finalizer
-    does is done and so that it no longer references any other objects. It does
-    $(I not) initiate a GC cycle or free any GC memory.
-  +/
+/********
```

Comment:
We really need to define a best practice for docstrings.
Generally I'm a fan of using only as many `*` / `+` as strictly necessary:

```
/++

+/
```

or

```
/**

*/
```

Reply:
Same here. Not a fan of lots of stars or stars at the beginning of each comment line.

Reply:
I was following @andralex's prior suggestion here: https://github.com/dlang/dmd/pull/6835#discussion_r158275983

> Let's use https://github.com/dlang/phobos/blob/master/std/string.d#L2254-L2267 for new code, thanks.

Anyway, if it's important, please make a PR to the style guide with preferred dos and don'ts

Reply:
https://github.com/dlang/dlang.org/pull/2129